src/app/file-reader/file-reader.component.html

```diff
@@ -1,16 +1,24 @@
-<div>
-  <label for="file-type-csv">Comma Separated Values Playlist (CSV|TXT):</label>
-  <input (change)="handleFileTypeChange($event)" type="radio" name="file-type-group"
-         id="file-type-csv" value="csv" checked="checked" [disabled]="fileInputDisabled">
+<div class="file-reader-wrapper">
+  <div class="option-pair">
+    <label for="file-type-csv">Comma Separated Values Playlist (CSV|TXT):</label>
+    <input (change)="handleFileTypeChange($event)" type="radio" name="file-type-group" id="file-type-csv" value="csv"
+      checked="checked" [disabled]="fileInputDisabled">
+  </div>
+
+  <div class="option-pair">
+    <label for="file-type-slacker">Slacker Playlist (XML):</label>
+    <input (change)="handleFileTypeChange($event)" type="radio" name="file-type-group" id="file-type-slacker"
+      value="slacker" [disabled]="fileInputDisabled">
+  </div>
 
-  <label for="file-type-slacker">Slacker Playlist (XML):</label>
-  <input (change)="handleFileTypeChange($event)" type="radio" name="file-type-group"
-         id="file-type-slacker" value="slacker" [disabled]="fileInputDisabled">
 </div>
 
 <p class="note">
-  <span *ngIf="fileType == 'csv'">CSV playlist must contain at least two named columns: TITLE, ARTIST.</span>
-  <span *ngIf="fileType == 'slacker'">Slacker playlist as returned by the https://slacker.livexlive.com/wsv1/playlists API.</span>
+  <span *ngIf="fileType == 'csv'">(CSV playlist must contain at least two named columns: <b>TITLE</b>,
+    <b>ARTIST</b>)</span> <br>
+    <a href="https://docs.google.com/spreadsheets/d/1bC0WZJ07E6LAKZx242uNW7IQ7e_aAHsmNkMyV13H9A4/edit?usp=sharing" target="_blank">Use template</a>
```

Comment:
Please remove the template link. I'd prefer the application code to be the single source of truth regarding the CSV structure. If columns are added someone will need to remember to update this template.

Reply:
Done.